.coveragerc

```diff
@@ -10,6 +10,7 @@ disable_warnings =
 # All paths are relative to each tox.ini
 omit =
     */test_bench.py
+    */test_e2e.py
```

Comment:
> We cannot run coverage for E2E tests as the code is non-local

But don't we still want to know if the `test_e2e.py` code has been executed or not ?